code/data_generator.py

```diff
@@ -0,0 +1,126 @@
+#%%
+
+import numpy as np 
+import random 
+import matplotlib.pyplot as plt
+from scipy import signal
+
+def data_simulator(amplitude, duration, mjd):
+
+    """ 
+    Generates random numbers between -10 and 10 every 4s for 4min
+    
+    inputs: 
+    amplitude: standard deviation of the stimulated data 
+    duration: in seconds
+    mjd: starting MJD in MJD
+    
+    outputs:
+    c1: time array with a point every 4s in mjd
+    nps: stimulated data 
+
+    """
+
+    # generate mjd every 4s for the duration in seconds 
+    duration_mjd = duration * 0.000675 #change the duration from seconds to MJD
```

Comment:
if we use duration_mjd, let's use duration_seconds instead of duration, it will be more readable